post.php

```diff
@@ -777,12 +777,8 @@
 
 // Display the header.
 echo $OUTPUT->header();
-echo $OUTPUT->heading(format_string($moodleoverflow->name), 2);
+echo $OUTPUT->heading('');
```

Comment:
why echo it at all?

Reply:
This explicitly says plugins should always use a more specific plugin renderer?

Reply:
Moreover, there are two more usages of $OUTPUT->heading(format_string($moodleoverflow->name), 2); 
One is displayed when deleting the post and should also be deleted